util/src/hash.rs

```diff
@@ -163,10 +161,7 @@ macro_rules! impl_hash {
 				// impl |= instead
 				// TODO: that's done now!
```

Comment:
not sure if it ever was, at least how i'm reading it.
we don't have any `impl BitOrAssign` for hash types, and it also isn't implemented for slices.